AIMSICD/src/main/res/values/translatable_strings.xml

```diff
@@ -318,7 +318,7 @@
 
 
     <!-- Section Main -->
-    <string name="help_main_neighboring_cells">Shows neighboring cells.</string>
+    <string name="help_main_neighboring_cells">Shows neighbouring cells.</string>
```

Comment:
There are at least 3 ways how you can get ciphering indicator in Android. Two are supported by AIMSICD and buggy, so it doesn't work. Still, change made by @larsgrefer  is good.